packages/manager/apps/pci-cold-archive/src/pages/containers/add-user/AddUser.page.tsx

```diff
@@ -19,13 +19,16 @@ import { TUser } from '@/api/data/users';
 import StepOneComponent from './StepOne.component';
 import StepTwoComponent from './StepTwo.component';
 import { useProductRegionsAvailability } from '@/api/hooks/useRegions';
+import { OBJECT_CONTAINER_USER_ROLE_ADMIN } from '@/constants';
 
 export default function AddUserToContainerPage() {
   const { t } = useTranslation('containers/add-user');
 
   const [stepUser, setStepUser] = useState(0);
   const [selectedUser, setSelectedUser] = useState<TUser>(null);
-  const [selectedRole, setSelectedRole] = useState<string>(null);
+  const [selectedRole, setSelectedRole] = useState<string>(
```

Comment:
To enhance typing, we could declare explicitly that it depends on the constants : 

```ts
    type TFoo = typeof OBJECT_CONTAINER_USER_ROLES[number]
```

Then, it should be safer than simple `string` primitive in components usage through props or state:

```ts
const [selectedRole, setSelectedRole] = useState<TFoo>(
```